src/indicators/acled_conflict/utils/map_conflict.R

```diff
@@ -24,7 +24,7 @@ map <- function(df_alerts, df_wrangled, df_raw, preview = FALSE) {
   df_map <- df_alerts |>
     dplyr$mutate(
       title = paste0(
-        "Conflict events since ",
+        "Reported events since ",
```

Comment:
Would they not still be conflict events?

Reply:
Putting as Reported events because they potentially cover just protests, riots, or other non-violent events in the dataset (although we only aggregate fatalities).